manifests/init.pp

```diff
@@ -51,6 +54,24 @@
     }].to_yaml %>')
   $policies_data = parseyaml($policies_yaml)
 
+  if $::rh_certificate_repo_ca_file {
```

Comment:
I agree and I think that's what @ohadlevy is refering to above. IMHO this should be moved to the params and because a class parameter should always be leading when set.